R/zzz.R

```diff
@@ -26,7 +27,14 @@ xportr_options_list <- list(
       "intervaldatetime"
     )
   ),
-  xportr.numeric_types = getOption("xportr.numeric_types", c("integer", "numeric", "num", "float"))
+  xportr.numeric_metadata_types = getOption(
+    "xportr.numeric_metadata_types",
+    c("integer", "numeric", "num", "float")
+  ),
+  xportr.numeric_types = getOption(
+    "xportr.numeric_types",
+    c("integer", "float", "posixct", "posixt", "time", "date")
+  )
```

Comment:
Should we also have `numeric` as part of this vector?
The absence of this causes us to coerce numeric to numeric.